src/data/zha.ts

```diff
@@ -40,6 +41,23 @@ export interface Neighbor {
   relationship: string;
 }
 
+export interface Route {
+  dest_nwk: string;
+  route_status: RouteStatus;
+  memory_constrained: boolean;
+  many_to_one: boolean;
+  route_record_required: boolean;
+  next_hop: string;
+}
+
+export enum RouteStatus {
```

Comment:
Should this be an enum? Or just a type that is a union of strings?

Reply:
Both are used in the codebase 🤷‍♂️